apps/test-quadrature64cpr/src/main.cpp

```diff
@@ -0,0 +1,60 @@
+#include "Logger.h"
+#include "Quadrature64CPR.h"
+
+// first pin is yellow wire
+// second pin is white wire
+constexpr Encoder::Quadrature64CPR::Config config = {PA_5, PA_6, NC, 0};
+Encoder::Quadrature64CPR encoder(config);
+
+// Synchronous testing
+void Updater();
+void Reader();
+Timer timer;
+
+constexpr auto PERIOD            = 500ms;
+constexpr bool USE_BLOCKING_TEST = true;
+
+// Print helper function
+void print(const std::string &str);
+Mutex print_mutex;
```

Comment:
I realized that I made Logger thread-safe so this thread-safe print function needed. You can replace print() with just printf(), it'll go through Logger and be thread-safe. I'll make this fix for the aeat test app

Reply:
sounds good let me know if i fixed it properly